quack_api/quack_api/Utilities/CommandLineProcess.cs

```diff
@@ -4,7 +4,20 @@
 using System.Threading.Tasks;
 
 namespace quack_api.Utilities
-{
+{ 
+    public class PythonPathNotFoundException : Exception
```

Comment:
I would not use Python in the name so only ```PathNotFoundException``` because the class CommandLineProcess is not only for Python.